uftrace.c

```diff
@@ -858,7 +861,17 @@ static int parse_option(struct uftrace_opts *opts, int key, char *arg)
 		break;
 
 	case OPT_disabled:
-		opts->disabled = true;
+		pr_use("'--disable' is deprecated, use --trace=on|off instead.\n");
+		opts->trace = TRACE_STATE_OFF;
+		break;
+
+	case OPT_trace:
+		if (!strcmp(arg, "on"))
+			opts->trace = TRACE_STATE_ON;
+		else if (!strcmp(arg, "off"))
+			opts->trace = TRACE_STATE_OFF;
+		else
+			pr_use("unknown tracing state: %s (ignoring..)\n", arg);
```

Comment:
It doesn't have to be included in this PR, but I think we can extend this option for later usage to control trace on/off with time duration. For example, we can support following usages.
```
# trace target program only for 1ms from the beginning.
$ uftrace record --trace=~1ms ./prog

# trace target program since 1ms elapsed from the beginning.
$ uftrace record --trace=1ms~ ./prog

# trace target program only from 1ms elapsed from the beginning to 3ms elapsed, then stop tracing.
# so target program is recorded during 2ms in total.
$ uftrace record --trace=1ms~3ms ./prog
```
Otherwise, we can support `-r`/`--time-range` option for this purpose instead for record time.

Reply:
Interesting, can you please file an issue separately?